### ISSUE TITLE
Need 11 character video id or the URL of the video. Got None.

### ISSUE THREAD

**Issue by muhahacz** [mopidy/mopidy-youtube#21]
Hello, i have got problem with playing from yt. Everytime when i tried to play m3u, mopidy shows message: `INFO Need 11 character video id or the URL of the video. Got None.`

```
mopidy --version
  Mopidy 0.19.5
mopidy deps
  Not found:
  none
```

Playlist file:

```
#EXTM3U
#EXTINF:-1, My song from YT
yt:http://www.youtube.com/watch?v=Njpw2PVb1c0
```

I'm able to play from youtube with this oneliner;

```
gst-launch-0.10 playbin2 uri=$(echo "http://www.youtube.com/watch?v=Njpw2PVb1c0" | python2 -c 'import pafy,sys;print(pafy.new("".join(sys.stdin)).getbestaudio().url)')
```

Thanks

**Comment by jodal** [comment]
Using Mopidy 1.1.0, Mopidy-YouTube 2.0.0 and pafy 0.3.74, playing an M3U with the contents you provided works nicely.

Please ask for the bug to be reopened if you still experience this issue.

**Comment by jcass77** [comment]
I get this error when searching youtube and then clicking on the returned results. Mopidy (1.1.1), Mopidy-Youtube (2.0.1), pafy (0.4.2).

It seems to be dependent on the web client somehow: does not work with Mopidy-MusicBox-Webclient (2.0.0) or Mopidy-Mobile (1.6.3), but works fine with Mopidy-HTTP-Kuechenradio (0.4.2).

Should this be re-opened?

**Comment by FacundoMainere** [comment]
I have this same problem with the same versions. It's a client issue?

**Comment by jcass77** [comment]
No I believe it is a bug in Mopidy-YouTube itself. A fix has been proposed and it should be resolved on way or the other soon.

**Comment by hamses33** [comment]
I've got the same error. 
Thats the Output

INFO     Need 11 character video id or the URL of the video. Got 
//com/playlist?list=PLeCg_YDclAETQHa8VyFUHKC_Ly0HUWUnq

WARNING  Track is not playable: yt:http://www.youtube.com
/playlist?list=PLeCg_YDclAETQHa8VyFUHKC_Ly0HUWUnq

**Comment by iambrianchan** [comment]
Was this fixed?

**Comment by urgemerge** [comment]
Found a workaround having exactly the same issue here running Mopidy (2.1.0), Mopidy-Youtube (2.0.2) and pafy (0.5.3.1). Youtube URLs are playable by searching and playing them using the Mopidy-Moped (0.7.1) web interface. I found out, running mopidy -vvv, that the called URi is a bit different:

`Sending track_playback_started to CoreListener: {'tl_track': TlTrack(tlid=90, track=Track(album=Album(images=[u'http://i.ytimg.com/vi/UBS4Gi1y_nc/hqdefault.jpg', u'http://i.ytimg.com/vi/UBS4Gi1y_nc/mqdefault.jpg'], name=u'YouTube'), comment=u'UBS4Gi1y_nc', length=633000, name=u'Aphex Twin - Windowlicker (official video) 1080p HD', uri='youtube:video/Aphex Twin - Windowlicker (official video) 1080p HD.UBS4Gi1y_nc'))} `

As a workaround it works for me to simply put the URI to my .m3u playlist:

`youtube:video/Aphex Twin - Windowlicker (official video) 1080p HD.UBS4Gi1y_nc`

Some shorter notation is also possible:

`yt:/.UBS4Gi1y_nc`

Cheers!
